litmus-portal/frontend/src/views/CreateWorkflow/VerifyCommit/styles.ts

```diff
@@ -149,5 +149,10 @@ const useStyles = makeStyles((theme: Theme) => ({
   buttomPad: {
     paddingBottom: theme.spacing(3.75),
   },
+  closeBtn: {
+    color: theme.palette.secondary.contrastText,
```

Comment:
Kindly avoid negative values for margins and try rearranging the divs. cc : @arkajyotiMukherjee

Reply:
@whitetiger1399 the litmus-ui modal is placing the button inside the content so this cannot be changed or the display is blocked by this button.

Reply:
Have updated at other places but here it's not working without negative margins because of the way the modal was built.

Reply:
It's better to raise the a PR to litmus-ui and avoid unwanted coding practices.
Can you track this change and make sure to remove it after updating litmus-ui @ishangupta-ds